src/app/app.module.ts

```diff
@@ -15,13 +15,15 @@ import {Phase3Module} from './phase3/phase3.module';
 import {Phase1Module} from './phase1/phase1.module';
 import {BrowserAnimationsModule} from '@angular/platform-browser/animations';
 import { UserConfirmationComponent } from './core/guards/user-confirmation/user-confirmation.component';
+import { PhaseTesterResponseModule } from './phase-tester-response/phase-tester-response.module';
 
 @NgModule({
   declarations: [
-    AppComponent, HeaderComponent, UserConfirmationComponent,
+    AppComponent, HeaderComponent, UserConfirmationComponent
```

Comment:
@KevinCJH Shall we standardize this by putting one component per line? Cause i think most of the other fields like `imports` and `exports` seem to work that way.

Reply:
Yup I think we should standardize one component per line